src/cuga/demo_tools/crm/crm_api/contacts.py

```
@@ -1,42 +1,42 @@
 @router.get("/{contact_id}", response_model=ContactResponse)
-def get_contact(contact_id: int, db: Session = Depends(get_db)):
+def get_contact(contact_id: int, db: sqlite3.Connection = Depends(get_db)):
     contact = contact_crud.get(db, contact_id)
     if not contact:
         raise HTTPException(status_code=404, detail="Contact not found")
     return contact
 
 
 @router.put("/{contact_id}", response_model=ContactResponse)
-def update_contact(contact_id: int, contact: ContactUpdate, db: Session = Depends(get_db)):
+def update_contact(contact_id: int, contact: ContactUpdate, db: sqlite3.Connection = Depends(get_db)):
```

Ruff (0.15.11)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)